kythe/rust/indexer/rustfmt.sh

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 The Kythe Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/sh
+
+for f in $(find src/ tests/ -name '*.rs')
+do
+    echo "Formatting $f";
+    rustfmt $f;
+done
```

Comment:
Given that a Cargo.toml exists, is this script redundant to `cargo fmt`?

Reply:
cargo-raze requires the Cargo.toml file to be in a separate subfolder. Because of that, `cargo fmt` does not function properly

Reply:
Looks like having the `lib` set in the Cargo.toml allows for running `cargo fmt` while inside of the `cargo` folder. I'll remove the script.

Reply:
Apparently it still skips some files. The script will need to stay for now.